tools/frr-reload.py

```diff
@@ -1961,6 +1962,44 @@ def compare_context_objects(newconf, running):
     return (lines_to_add, lines_to_del)
 
 
+class LogFmtFormatter(logging.Formatter):
+    def format(self, record):
```

Comment:
I would definitively add a doc string that summarizes what this does, something like:

```suggestion
    def format(self, record: logging.LogRecord) -> str:
    """
    Creates log messages with key=value pairs, such as ...
    """
```

(and also, add type annotations)

Reply:
For example the `IndentingFormatter` in the `logging.py` module has:

```python
class IndentingFormatter(logging.Formatter):
    default_time_format = "%Y-%m-%dT%H:%M:%S"
...

    def format(self, record: logging.LogRecord) -> str:
        """
        Calls the standard formatter, but will indent all of the log message
        lines by our current indentation level.
        """
        formatted = super().format(record)
...
```